src/bindings/dom.js

```diff
@@ -89,7 +90,11 @@ function translateElements(view, elements) {
 function applyTranslations(view, elems, translations) {
   disconnect(view, null, true);
   for (let i = 0; i < elems.length; i++) {
-    overlayElement(elems[i], translations[i]);
+    try {
+      overlayElement(elems[i], translations[i]);
+    } catch (e) {
+      console.log(e);
+    }
```

Comment:
that means that if overlayElement fails, we don't break the rest of UI. Not sure if we should land it or make overlayElement more robust.